apps/wallet/src/ui/app/pages/home/tokens/TokensDetails.tsx

```diff
@@ -159,11 +165,20 @@ export function TokenDetails() {
 
     const onSendClick = () => {
         if (!activeAccount?.isLocked) {
-            const destination = coinBalance?.coinType
-                ? `/send?${new URLSearchParams({ type: coinBalance?.coinType }).toString()}`
-                : '/send';
+            if (activeAccount) {
```

Comment:
should we add this check to outer if clause
`if (activeAccount && !activeAccount?.isLocked) {`